File: EmprestimoApi/EmprestimoApi/Models/Cliente.cs
using System.ComponentModel.DataAnnotations;

namespace EmprestimoApi.Models
{
    public class Cliente
    {
        [Key]
        public int Id { get; set; }
        public string Nome { get; set; }
        public int Idade { get; set; }
        public double Score { get; set; }
        public double SaldoAtual { get; set; }
        public Emprestimo? Emprestimo { get; set; }
    }
}

File: Emprestimoapp/src/app/cliente/models/cliente.ts
import { Emprestimo } from './../../emprestimo/models/emprestimo';
export interface Cliente{
    clienteId: number;
    nome:string;
    idade:number;
    score:number;
    cpf: string;
    saldoAtual:number;
    emprestimo:Emprestimo
}